src/ServicePulse.Host/Owin/EmbeddedFileFinder.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using ServicePulse.Host.Owin.Microsoft;
+
+namespace ServicePulse.Host.Owin
+{
+    public static class EmbeddedFileFinder
+    {
+        private static readonly Assembly Assembly = Assembly.GetExecutingAssembly();
```

Comment:
Is this what's really wanted? Is `Assembly.GetExecutingAssembly();` returning the assembly that contains this class? which makes so that embedded files will be searched only in this assembly.

Reply:
We want to search inside the ServicePulse.Host.exe assembly only, so I think this works fine. 

Or maybe I don't completely understand your concern, @mauroservienti ?

Reply:
I think that's the expected behaviour. This class is in the host assembly, and all the js, html, and css files are bunclded in the host assembly (bleurgh).

Reply:
`private static readonly Assembly Assembly = typeof(EmbeddedFileFinder).Assembly;`  is slightly more robust if this code is somehow suddenly not hosted in `ServicePulse.Host.exe`, but the self-hosting nature of ServicePulse makes that very unlikely?

Reply:
> We want to search inside the ServicePulse.Host.exe assembly only, so I think this works fine.

This answers my question, @janovesk. Thanks.